chart/files/pod-template-file.kubernetes-helm-yaml

```diff
@@ -45,6 +45,7 @@ spec:
   {{- end }}
   containers:
     - args: []
+      securityContext: {{- omit .Values.podSecurity.pod_template.containerSecurityContext "enabled" | default (.Values.podSecurity.default.containerSecurityContext) | toYaml | nindent 8 }}
```

Comment:
Instead of this `omit` pattern, would this work instead? I think it will and is more intuitive imo.

values.yaml:
```
podSecurity.pod_template.containerSecurityContext: {}
```

and:

```suggestion
      securityContext: {{- .Values.podSecurity.pod_template.containerSecurityContext | default (.Values.podSecurity.default.containerSecurityContext) | toYaml | nindent 8 }}
```

Reply:
The template was amended to reflect the change.